plugins/dice.js

```diff
@@ -12,7 +25,9 @@ module.exports = (function(){
 			aResults = [];
 		for (let i = 0; i < throws; i++){
 			let roll = Math.floor((Math.random() * sides) + 1);
-			result = result + roll;
+			result += roll;
+			if (roll === 1) roll = '`' + roll + '`';
```

Comment:
similar to above, I try to avoid bracket-less if statements except in very small circumstances.  Here is some reasoning: http://stackoverflow.com/questions/4797286/are-curly-braces-necessary-in-one-line-statements-in-javascript